Makefile

```diff
@@ -48,23 +48,12 @@ dist/lib.js: dist dist/d3.js $(LIBS)
 		lib/d3.keybinding.js \
 		lib/d3.trigger.js \
 		lib/d3-compat.js \
-		lib/draw/leaflet.draw-src.js \
-		lib/drag/leaflet.drag.js \
 		lib/codemirror/lib/codemirror.js \
 		lib/codemirror/mode/javascript/javascript.js > dist/lib.js
 
 dist/delegate.js: src/delegate.js
 	$(BROWSERIFY)  src/delegate.js > dist/delegate.js
 
-lib/mapbox.js/latest:
-	mkdir -p lib/mapbox.js/latest
-
-lib/mapbox.js/latest/mapbox.js: node_modules/mapbox.js/*
-	$(BROWSERIFY) node_modules/mapbox.js > lib/mapbox.js/latest/mapbox.js
-	$(UGLIFY) -o lib/mapbox.js/latest/mapbox.js lib/mapbox.js/latest/mapbox.js
-	cp -r node_modules/mapbox.js/theme/images/ lib/mapbox.js/latest/images/
-	$(CLEANCSS) -o lib/mapbox.js/latest/mapbox.css --skip-rebase node_modules/mapbox.js/theme/style.css
-
```

Comment:
Will there be any need to replace these resources in the makefile with gl.js equivalents?

Reply:
I am not sure why mapbox.js was being included with the build versus loaded from a CDN.  This PR just loads mapboxGL and its CSS from the CDN.